main/openchange/src/EBox/OpenChange.pm

```diff
@@ -686,6 +689,33 @@ sub setProvisioned
     $self->set_state($state);
 }
 
+# Method: usersNum
+#
+#     Return the number of users using OpenChange, that is, the MAPI users
+#
+# Returns:
+#
+#     Int - 0 if not provisioned, a natural number otherwise
+#
+sub usersNum
```

Comment:
I assume that the mailbox is only created on the first connection. Why we count mailboxes instead of provisioned users?